x/bank/types/msgs.go

```diff
@@ -176,3 +179,28 @@ func ValidateInputsOutputs(input Input, outputs []Output) error {
 
 	return nil
 }
+
+// Route returns the MsgUpdateParams message route.
+func (msg MsgUpdateParams) Route() string { return ModuleName }
+
+// Type returns the MsgUpdateParams message type.
+func (msg MsgUpdateParams) Type() string { return TypeMsgUpdateParams }
```

Comment:
I don't think we need to implement the legacy APIs for this new message.

```suggestion
```

Reply:
Removed!